spec/support/helpers/features_helper.rb

```diff
@@ -1,8 +1,21 @@
 # frozen_string_literal: true
 
 module FeaturesHelper
-  def signin_spo_user(name = 'MOIC_POM')
-    mock_sso_response(name, [SsoIdentity::SPO_ROLE])
+  def signin_spo_user(prison = ['LEI'])
+    mock_sso_response('MOIC_POM', [SsoIdentity::SPO_ROLE], prison)
+  end
+
+  def stub_signin_spo(pom, _prison = ['LEI'])
+    stub_auth_token
+    signin_spo_user(_prison = ['LEI'])
```

Comment:
I think the issue is that this isn't really valid ruby syntax. It's assigning a temporary unused variable